PasteRack script

spdegabrielle opened the pull request:
simple script that launches PasteRack

spdegabrielle committed:
PasteRack script

Metaxal commented on PasteRack.rkt:
```diff
@@ -0,0 +1,10 @@
+(provide item-callback)
+(define (item-callback str)
+  (send-url "http://pasterack.org/")
+  str)
```
Make it return #f instead of str so that the file is not modified and I'll approve the push request.
See http://docs.racket-lang.org/script-plugin/index.html#%28part._.The__rkt_file%29

spdegabrielle committed:
Update PasteRack.rkt

spdegabrielle commented:
Done!

Metaxal commented:
Looks good, but can you place the two files in the 'examples' directory with the other scripts, and also maybe follow the name convention (probably "pasterack.rkt" would do)?

I'm not used to merging others' pull request, so I'm not sure if I should first accept your patch and then modify it myself or ask you to do the changes... sorry if it's cumbersome.

spdegabrielle committed:
Rename PasteRack.rkt to examples/pasterack.rkt

spdegabrielle committed:
Rename PasteRack.rktd to examples/pasterack.rktd

spdegabrielle commented:
I'll admit I was confused they ended up there. I clicked 'New Script' and that was the default position.  Is that a bug?
 
Anyway I've renamed and moved the into /examples/. 

BTW - should examples renamed /scripts/ ?

spdegabrielle commented:
> Is that a bug?
Its not a bug - I retested and it works fine. I must have clicked up a folder.
- I just submitted a PR for tweet.rkt clicking 'New Script..." put it in the right folder.

spdegabrielle commented:
> I'm not used to merging others' pull request, so I'm not sure if I should first accept your patch and then modify it myself or ask you to do the changes... sorry if it's cumbersome.

I'm new to this too :)

I've made small PRs to DrRacket (documentation) and Robby got me to do the changes. I think this is the right approach or otherwise it would become too onerous for the maintainer.

Kind regards, 
Stephen

Metaxal commented:
Ok, good to know.

Thanks for the new script!
